drivers/afe/ad4110/ad4110.c

```diff
@@ -420,6 +420,9 @@ int32_t ad4110_spi_int_reg_write(struct ad4110_dev *dev,
 		buf[data_size] = ad4110_compute_crc8(&buf[0], data_size);
 	} else
 		buf_size = data_size;
+
+	return spi_write_and_read(dev->spi_dev, buf, buf_size);
+}
```

Comment:
This change is not related with this commit.
Code after this change looks:
![image](https://user-images.githubusercontent.com/16192750/140931603-aed8c81f-e57c-48bc-be29-33b488405506.png)

Reply:
Each individual commit shouldn't break anything and compile.